src/components/popups/linkrequestpopup.vue

```diff
@@ -16,16 +17,22 @@
             {{ $t('operations:link.request', {appName: incoming.appName, origin: incoming.origin, chain: incoming.chain }) }} &#10068;
         </div>
         <br>
+        <div v-if="existing.length>0">
+            {{ $t('operations:link.request_fresh', {chain: incoming.chain }) }}
+        </div>
+        <br>
         <AccountSelect
+            v-if="incoming.chain"
```

Comment:
What happens if incoming has no chain?

Reply:
it always does :) it could be removed i think...probably left over from some iteration